scripts/ci-entrypoint.sh

```diff
@@ -132,91 +132,104 @@ create_cluster() {
 }
 
 install_addons() {
```

Comment:
this func is getting a little big, might be time to split up the each different operation in its own func?

Reply:
that would also allow retrying each block independently instead of having to keep track of `CIDRS_RETRIEVED`, `CALICO_INSTALLED`, etc.

Reply:
Depending on what set of kubectl ops we're making a "transaction" of we still will need to do this type of accounting, but if splitting up into multiple funcs is more maintainable we can easily do that.

Reply:
I've tried to incorporate your suggestion, and I do think this makes things more readable and maintainable, PTAL, thanks!